scripts/secret_decoder_ring.py

```diff
@@ -2,50 +2,47 @@
 """
 Script to encode/decode the IDs that galaxy exposes to users and admins.
```

Comment:
FWIW this functionality is duplicated in helper.py.

Reply:
I noticed that also - I just kept both for backward compatibility.

Reply:
Huh. I didn't know about that one. I shouldn't've added secret-decoder-ring when it was already existing.

Reply:
Could we remove secret decoder and replace it with a symlink to helper? Any usage of this script is assuredly interactive, so it's not like anyone is going to have flags hard coded into their workflows that they'd have to update.

Reply:
Can we merge this as is and follow up with that in dev?

Reply:
oh yeah, totally, sorry didn't mean that as a PR blocking comment :)